### DISCUSSION TITLE
Voltage Regulator for Power flow

### DISCUSSION THREAD

**Discussion by ConnorG51** [PowerGridModel/.github#22] [Q&A] [answered]
Is there a voltage regulator component that can be added to the network? 

And if not, do the transformers have on-load tap changers (OLTC) to potentially be a substitute for the voltage regulator?

**Comment by TonyXiang8787** [comment, marked as answer]
Hi @ConnorG51,

Thanks for your question. We are working on voltage regulator in this issue https://github.com/PowerGridModel/power-grid-model/issues/74. It is planned in the road map in Q1 2024.

**Reply by petersalemink95** [reply]
@ConnorG51 To update you on the progress: due to some delays https://github.com/PowerGridModel/power-grid-model/issues/74 will be expected in Q2 instead of Q1. We are working on it!

**Comment by marialame121-blip** [comment]
Yes, voltage regulators can be added to a network, especially in long distribution feeders to control voltage levels.

But many substations already manage voltage using On-Load Tap Changers (OLTC) inside power transformers. OLTCs adjust the transformer ratio while the unit is energized, helping maintain stable voltage without interrupting supply.

Because of this, OLTC-equipped transformers often reduce the need for separate regulators at the substation level. Proper maintenance of these systems is important, which is why utilities often work with OLTC specialists like [EMR Global](https://emr.global/) for components and service.
